cips/cip-tx-limits.md

```diff
@@ -0,0 +1,44 @@
+| cip | TBD(27) |
+| - | - |
+| title | Block limits for number of PFBs/non-PFBs and transaction size |
```

Comment:
We discussed this in all core devs but IMO there wasn't a clear decision made in the call. I think we could split this into two CIPs:

1. Block limits for PFBs / non-PFBs 
2. Limit transaction size

Reply:
I think @cmwaters explanation is that these both impact how a block is constructed, so was okay to leave it as 1 CIP.

Reply:
I'm also happy to scrap this and have it split to 2 CIPs by authors of the features.